Docs/articles/Kernel/CGS.md

```diff
@@ -72,38 +75,34 @@ namespace GraphicTest
-
                 /* A LimeGreen rectangle */
                 pen.Color = Color.LimeGreen;
                 canvas.DrawRectangle(pen, 450, 450, 80, 60);
 
-                Console.ReadKey();
-
-                Stop();
+                Sys.Power.Shutdown();
             }
             catch (Exception e)
             {
                 mDebugger.Send("Exception occurred: " + e.Message);
-                mDebugger.Send(e.Message);
-                Stop();
+                Sys.Power.Shutdown();
             }
         }
     }
 }
 ```
-# Limitation of the current implementation
-
-1. Only the Bochs Graphic Adapter is actually supported; this means that CGS will work only on Bochs, QEMU and VirtualBox. CGS does not make any assumption on the underlying hardware, simple a VGA driver should be adapted to use it. No change to user code should be required to use another driver
-2. Only 32 bit color depth is actually supported, the API provides methods to set a resolution with 24, 16, 8 and 4 bit but the low level Bochs driver has not yet implemented them
-3. CGS does not permits yet to do basic operations that would permit to fulfill its promise to be the basic block from which a "Widget Toolkit" could be derived for example these methods should be added:
-    DrawFilledRectangle((Pen pen, int x_start, int y_start,int width, int height)
-    void DrawImage(Image image, int x, int y) Canvas has already defined this method but it is not yet implemented
-    void DrawString(String string, Font font, Brush brush, int x, int y) Canvas has already defined this method but it is not yet implemented
-    .Net System.Drawing has overload of these methods taking a float to evaluate if they are really needed for CGS, they are defined but for now no implementation is provided
-    A double buffering strategy could be implemented to make it faster
-    This is more a limitation of the Bochs driver when the Graphic Mode is set there is no more a way to return to text mode
-    CGS / Graphic Mode interacts badly with the Kernel.Stop method: Bochs does not exit cleanly. You must use Sys.Power.Shutdown to shut down correctly your computer.
+# Limitations of the current implementation
+
+1. Only the Bochs Graphic Adapter (VBE) is actually supported; this means that CGS will work only on Bochs, QEMU and VirtualBox. CGS does not make any assumption on the underlying hardware, simple a VGA driver should be adapted to use it. No change to user code should be required to use another driver.
+
+2. Only 32 bit color depth is actually supported, the API provides methods to set a resolution with 24, 16, 8 and 4 bit but the low level Bochs driver has not yet implemented them.
+
+3. CGS does not permits yet to do basic operations that would permit to fulfill its promise to be the basic block from which a "Widget Toolkit" could be derived from. For example these methods should be added:
+    - void DrawFilledRectangle(Pen pen, int x_start, int y_start, int width, int height)
```

Comment:
At least DrawImage and DrawString exist. Maybe add them to the article?

Reply:
They aren't implemented yet right?

Reply:
Also they're already in the third section, I'll add them to the list of functions at the top.

Reply:
They should be implemented, so they can be removed here. I just checked `DrawFilledRectangle` is also implemented, so this point can be removed.